tests/flow/new_react/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -865,11 +865,7 @@ var ReactClass = React.createClass({
   render: function(): any {
     // Any state access here seems to make state any
     this.state;
-    return (
-      <div>
-        {this.state.bar.qux}
-      </div>
-    );
+    return <div>{this.state.bar.qux}</div>;
```

Comment:
This was intentional, in our codebase at Geckoboard we generally put single expressions in a tag on a single line (if they fit).

I'it's easy to tweak this to never inline if an element contains expressions.